### DISCUSSION TITLE
Where can I find ebuspicloader for windows for download?

### DISCUSSION THREAD

**Discussion by Juergen2020** [john30/ebusd#986] [Q&A] [answered]
the download link on ebusd.eu is broken.
on linux I've the problem to find the usb-device. if I connect the ebus-adapter to the linux system, I can't find any device. I don't know the name. 

thx 
jürgen

**Comment by Koky05** [comment, marked as answer]
It just missing /v31/ in link.

https://adapter.ebusd.eu/v31/firmware/ebuspicloader-windows.zip

**Reply by Juergen2020** [reply]
thx
